features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ExploreMapFragment.kt

```diff
@@ -192,82 +177,98 @@ class ExploreMapFragment: SupportMapFragment() {
 
     private fun addMarkerOnCurrentPosition() {
         val bitmap = getBitmapFromDrawable(R.drawable.user_location_map_marker)
-        currentLocationMarker = markerCollection?.addMarker(MarkerOptions().apply {
+        currentLocationMarker = googleMap?.addMarker(MarkerOptions().apply {
```

Comment:
There is no need to save the current location marker in the same collection as the merchant/atm markers. This simplifies managing other markers like handling clicks and calculating the difference to render.